users/twschum/twschum.c

```diff
@@ -0,0 +1,270 @@
+#include "twschum.h"
+
+// util for detecting simultanious shifts
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```

Comment:
As above.  This doesn't need to be here, since we have `MOD_MASK_CTRL` defined globally.